roadmap.md

```diff
@@ -1,110 +1,163 @@
-.NET Core Roadmap
-=================
-
-.NET Core is a general purpose, modular, cross-platform and open source implementation of .NET. It includes a runtime, framework libraries, compilers and tools that support a variety of chip and OS targets. These components can be used together or separately.
+# .NET Core Roadmap
+
+Our primary focus is to get the .NET Core 1.0 tooling to RTM quality for the
+Visual Studio 2017 RTM ([please try the 2017 RC update][vs2017-rc] we shipped
+today with improvements in this area).
+
+However, we are also starting to think of the next version of the runtime.
+Because of this, if you follow our repositories you will start to see .NET Core
+2.0 versioning. While we are still in the process of planning the next release,
+there are some big themes you will identify in the early work on GitHub:
+
+[vs2017-rc]: https://blogs.msdn.microsoft.com/dotnet/2016/12/12/updating-visual-studio-2017-rc-net-core-tooling-improvements/
+
+* **.NET Standard 2.0**. Having a large set of common APIs that are consistent
+  across .NET Framework, .NET Core and Xamarin making it easier to share code
+  across those platforms and making it easier for package authors to build NuGet
+  packages. .NET Core gain over 5,000 APIs from .NET Framework as part of this
+  work. You can follow this work in the
+  [dotnet/standard](https://github.com/dotnet/standard) repo.
+
+* **.NET Core Tooling**. Evolve the tooling aligned with the next .NET Core
+  runtime release. This will include tooling for choosing which .NET Core
+  version to target, to change the version of .NET Core for existing projects,
+  full package IntelliSense in .csproj files and more. You can follow this work
+  in the
+  [dotnet/roslyn-project-system](https://github.com/dotnet/roslyn-project-system),
+  [dotnet/sdk](https://github.com/dotnet/sdk), and
+  [microsoft/msbuild](https://github.com/microsoft/msbuild) repos.
+
+* **Performance**. Continue to make the performance of building .NET Core
+  applications faster, especially in the inner loop. This is the cycle of
+  changing the source code and then restarting the application and making that
+  as fast as possible. You can follow this work
+  [dotnet/corefxlab](https://github.com/dotnet/corefxlab/blob/master/docs/roadmap.md)
```

Comment:
The corefxlab work is steady state performance, not about making the inner loop faster that this paragraph talks about.